exercises/two-bucket/src/example.h

```diff
@@ -0,0 +1,24 @@
+#ifndef TWO_BUCKET_H
+#define TWO_BUCKET_H
+
+#include <stdbool.h>
+
+typedef enum {
+   bucket_id_1,
+   bucket_id_2
+} bucket_id_t;
+
+typedef unsigned int bucket_volume_t;
+
+typedef struct {
+   bool possible;
+   int move_count;
+   bucket_id_t goal_bucket;
+   bucket_volume_t other_bucket_volume;
```

Comment:
The naming of `other_bucket_volume` and `goal_volume` is confusing to me. `volume` seems like a better descriptor for the size of the bucket and not the amount of water in the bucket.

Consider using something like `other_bucket_remaining_liters` or `goal_liters`.